scripts/release.py

```diff
@@ -268,17 +268,19 @@ def publish_release(self):
             else:
                 raise
 
-    def update_milestones(self):
-        """Create GitHub release page for tag"""
+    def update_milestones(self, next_version=None):
```

Comment:
This class is designed to complete one release and setup the next one. It was not designed to support the setup of multiple new releases/milestones. If you look at some of the other attributes defined in __init__ you'll see they are in line with a current and next release.

I think it would be better to define the date when you create the release object, with it defaulting to self.tag.next_revision, and call the method when you want to set the milestone in GitHub.